src/templates/dandelion/components/TokenSelector/TokenSelectorInstance.js

```diff
@@ -36,20 +36,30 @@ const TokenSelectorInstance = React.memo(function TokenSelectorInstance({
           {symbol}
         </span>
       )}
-      {name && (
-        <span
-          css={`
-            max-width: 110px;
-            margin-right: ${1 * GU}px;
-            overflow: hidden;
-            text-overflow: ellipsis;
-          `}
-        >
-          ({name})
-        </span>
-      )}
-      {!addressesEqual(address, ETHER_TOKEN_FAKE_ADDRESS) &&
-        shortenAddress(address)}
+      <div>
+        (
+        {name && (
```

Comment:
Small syntax nitpick: I think this can be combined with the previous brace (surprised prettier didn't do this).

Reply:
The thing here is that it's a literal `(` and not the one used for enclosing components (Actually prettier auto-saves it this way)